core/Db/Schema/Mysql.php

```diff
@@ -299,6 +299,21 @@ public function getTablesCreateSql()
                                       ) ENGINE=$engine DEFAULT CHARSET=utf8
             ",
 
+            'archive_invalidations' => "CREATE TABLE `{$prefixTables}archive_invalidations` (
+                                            idinvalidation BIGINT UNSIGNED NOT NULL AUTO_INCREMENT,
+                                            idarchive INTEGER UNSIGNED NULL,
```

Comment:
btw is it supposed to be nullable?

Reply:
Yes, this is in case it's for an archive that does not exist. Eg, if we invalidate a day archive, but there is no week, month or year (because here are no visits for them or it hasn't been processed), we still want to process them. In this case we have to set idarchive to NULL.